btc/src/main/kotlin/com/d3/btc/handler/SetAccountDetailHandler.kt

```diff
@@ -0,0 +1,21 @@
+package com.d3.btc.handler
+
+import iroha.protocol.Commands
+
+/**
+ * SetAccountDetail command handler interface
+ */
+interface SetAccountDetailHandler {
+    /**
+     * Handling logic
+     * @param command - command to handle
+     */
+    fun handle(command: Commands.SetAccountDetail)
+
+    /**
+     * Filter of commands. Only filtered commands will be handled
```

Comment:
`Only filtered commands will be handled` that is not true for now 😆 
You can refactor the method to `isProcessable` (if you need it accessable at all) and use it inside `handle`